packages/tools/src/tools/annotation/PlanarFreehandROITool.ts

```diff
@@ -158,9 +158,11 @@ class PlanarFreehandROITool extends AnnotationTool {
         // be infinite as the lines become very computationally expensive to draw.
         subPixelResolution: 4,
         interpolation: {
+          editEnabled: false, // used for edit only
```

Comment:
I know you have put excellent docs on the .md file, can you copy paste here for each of these params in summary, and the full thing above the class so that it appears in the api doc?

Reply:
what is `editEnabled` vs `enabled`? maybe we choose another name? `editMode`?

Reply:
I think `editEnabled` means whether to interpolate on edit or not, hmmm, maybe we go with something else like `interpolateOnEdit`?